### DISCUSSION TITLE
Firefly self hosted server change port

### DISCUSSION THREAD

**Discussion by ghost** [firefly-iii/firefly-iii#6780] [Firefly III]
Hello, is there a way for me to change the port for firefly? Instead of using http://IP-ADDRESS/firefly-iii, I would like to use a different port i.e http://IP-ADDRESS:8000/firefly-iii, I have followed all necessary steps for installing the server to include the FAQ. I have also set my Apache server to listen on port 8000,

**Comment by JC5** [comment]
That should do it, yep.

**Reply by ghost** [reply]
Is there something in a config file I need to change in /var/www/firefly directory?

**Reply by JC5** [reply]
No, you need to configure Apache.
